apps/evm/lib/evm/address.ex

```diff
@@ -38,7 +39,7 @@ defmodule EVM.Address do
   @doc """
   Returns an address given an address and a nonce.
   """
-  @spec new(integer(), integer()) :: binary()
+  @spec new(binary(), integer()) :: t()
```

Comment:
binary() -> t() ?

Reply:
The type here is an `t :: <<_::160>>`, which represents an Ethereum address. This is to make the type more specific.